src/app/faqs/faqs.component.ts

```diff
@@ -11,7 +11,8 @@ interface FaqContent {
 @Component({
   selector: 'app-faqs',
   templateUrl: './faqs.component.html',
-  styleUrls: ['./faqs.component.scss']
+  styleUrls: ['./faqs.component.scss'],
+  encapsulation: ViewEncapsulation.None // allow styles to apply to innerHTML,
```

Comment:
What issue(s) are we solving with this?

Reply:
Since some of the text contain `<a>` we need to apply some stylings to it. To make the [styles](https://stackoverflow.com/a/44215795) work we have to use it.

Reply:
Aren't we applying the styling to the parent/outer element though i.e. are we using something like `<a class="" ...` in the `innerHTML`?

Reply:
Yes. This is resolved once we move the styling to the `styles`